tests/baselines/reference/user/bluebird.log

```diff
@@ -299,7 +299,6 @@ node_modules/bluebird/js/release/using.js(78,20): error TS2339: Property 'doDisp
 node_modules/bluebird/js/release/using.js(97,23): error TS2339: Property 'data' does not exist on type 'FunctionDisposer'.
 node_modules/bluebird/js/release/using.js(131,72): error TS2339: Property 'classString' does not exist on type 'typeof ret'.
 node_modules/bluebird/js/release/using.js(223,15): error TS2350: Only a void function can be called with the 'new' keyword.
-node_modules/bluebird/js/release/util.js(200,32): error TS2339: Property 'foo' does not exist on type 'FakeConstructor'.
```

Comment:
better (?) type checking leads us to find an `any` for `FakeConstructor.prototype`, meaning that any property accesses are allowed:

```js
function toFastProperties(obj) {
  function FakeConstructor() { }
  FakeConstructor.prototype = obj;
  var receiver = new FakeConstructor();
  return receiver.foo;
}
```